src/librustdoc/html/static/main.js

```diff
@@ -1281,7 +1281,7 @@ function hideThemeButtonState() {
         if (currentType) {
             currentType = currentType.getElementsByClassName("rust")[0];
             if (currentType) {
-                currentType.classList.forEach(function(item) {
+                onEachLazy(currentType.classList, function(item) {
```

Comment:
Oh, it's not DOM. Ok, I see the issue. It's simply that IE11 array didn't implement that. How weird... Anyway:

```suggestion
                onEach(currentType.classList, function(item) {
```

no need for the `lazy` part here.

Reply:
IE11 does implement forEach on regular arrays, but this is not an array. It is a [`DOMTokenList`](https://caniuse.com/mdn-api_domtokenlist_foreach).

Reply:
You're absolutely right, my bad!